configuration_go/abstr/kubernetes/thanos/store/store.go

```diff
@@ -119,7 +119,6 @@ func NewStore() *StoreStatefulSet {
 			Replicas:             1,
 			PodResources:         k8sutil.NewResourcesRequirements("500m", "1", "200Mi", "400Mi"),
 			Affinity:             *k8sutil.NewAntiAffinity(nil, labelSelectors),
-			SecurityContext:      k8sutil.GetDefaultSecurityContext(),
```

Comment:
Why are we removing this?

Reply:
I had issues for statfulsets having this. The user group is considered invalid and the PV could not be mounted... I don't know where this policy comes from though.